ste/uploader-blobFS.go

```diff
@@ -136,6 +185,9 @@ func (u *blobFSUploader) RemoteFileExists() (bool, error) {
 }
 
 func (u *blobFSUploader) Prologue(state common.PrologueState) {
+	//Start bfsTracker before anything else, but only once.
+	go bfsTrackerOnce.Do(bfsTrackerRoutine)
```

Comment:
I have the opposite view to Ze on this one ;-)  I'm not comfortable with us just running one Go routine for all uploaders. It's architecturally different from everything else that we do, seems less understandable, and harder to reason about failure scenarios.  For downloads, we already run one extra GR per file; I don't see why we couldn't do the same for uploads.